test/PoseidonGoldenTests/GoldenTestsRunCommands.hs

```diff
@@ -848,6 +846,7 @@ testPipelineForge testDir checkFilePath = do
     runAndChecksumFiles checkFilePath testDir action17 "forge" [
           "forge" </> "ForgePac17" </> "ForgePac17.janno"
         ]
+    return ()
```

Comment:
Why is this return statement necessary here now?

Reply:
ha, good catch. I guess I removed it to stop some compiler errors at some point, but clearly not necessary. I removed it.